packages/cli-packages/storybook/package.json

```diff
@@ -19,11 +19,6 @@
     "build:watch": "nodemon --watch src --ext \"js,jsx,ts,tsx\" --ignore dist --exec \"yarn build\"",
     "prepublishOnly": "NODE_ENV=production yarn build"
   },
-  "jest": {
-    "testPathIgnorePatterns": [
-      "/dist/"
-    ]
-  },
```

Comment:
I removed these Jest related config options here and in a few other places in the spirit of cleaning up because they weren't being used